test/operators.jl

```diff
@@ -97,3 +97,15 @@ end
     B = 3 .> [1 -1 5] .> 0
     @test B == [true false false]
 end
+
+Base.:(<)(x::Type, y::Type) = (x <: y) & (x != y)
```

Comment:
I think this should be wrapped in a custom type e.g. `struct SortType x end`. While no other test should be relying on this, there could be future interference should another test want to do something similar.